src/sass/modules/_m-form-elements.scss

```diff
@@ -75,9 +75,9 @@ button.form-button-disabled {
   // See 18F/web-design-standards#1490
   // This fixes things in the meantime.
   [type="checkbox"] {
-    height: auto !important;
+    height: 1.8rem !important;
```

Comment:
@raquelromano-gov [these instructions](https://github.com/department-of-veterans-affairs/vets.gov-team/blob/3503c5b9fd936d83364f28a9ffe5bed2a2a7ed28/Products/Rx%20Refills/TestRxSMOnStaging.md) may be helpful. They are about testing on staging, but if you have vets-api set up locally, the same user can be used locally as well.